client.go

```diff
@@ -23,6 +23,20 @@ type Client struct {
 // NewClient initializes connections and authenticates with the
 // backend service with the credentials provided as arguments.
 func NewClient(clientID, clientSecret string) (*Client, error) {
+	cClient, err := newClient(C.AE_LICENSE_SEARCH, clientID, clientSecret)
+	if err != nil {
+		return nil, err
+	}
+	return &Client{
+		c: cClient,
+	}, nil
+}
+
+func (x *Client) getCClient() *C.AE_Client {
+	return x.c
+}
+
+func newClient(typ C.AE_ClientType, clientID, clientSecret string) (*C.AE_Client, error) {
```

Comment:
```suggestion
func newClient(type C.AE_ClientType, clientID, clientSecret string) (*C.AE_Client, error) {
```

Is that intentional or typo?

Reply:
intentional, type is a keyword in go.